packages/client/src/views/CorrectionForm/utils.ts

```diff
@@ -99,6 +100,28 @@ export function isCorrection(declaration: IDeclaration) {
   )
 }
 
+export function motherDoesNotExistAndStateIsMother(
+  declaration: IDeclaration,
+  activeState: string
+) {
+  return (
+    !Boolean(declaration.data.mother.detailsExist) &&
+    activeState === 'mother' &&
+    declaration.data.registration.informantType.value !== 'MOTHER'
+  )
+}
+
+export function fatherDoesNotExistAndStateIsFather(
+  declaration: IDeclaration,
+  activeState: string
+) {
+  return (
+    !Boolean(declaration.data.father.detailsExist) &&
+    activeState === 'father' &&
+    declaration.data.registration.informantType.value !== 'FATHER'
```

Comment:
Notice that in the reference code, we also check for `selectedContactType`. I think in the UI we want to enable adding attachments also to the mother and father when they are the contact person.

https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/forms/utils.ts#L531-L544